charts/cf-runtime/values-rootless.yaml

```diff
@@ -19,7 +19,7 @@ volumeProvisioner:
 runtime:
   dind:
     image:
-      tag: 26.1.4-1.28.10-rootless
+      tag: 28.1.1-3.0.1-rootless
       digest: sha256:59dfc004eb22a8f09c8a3d585271a055af9df4591ab815bca418c24a2077f5c8
```

Comment:
@vitalii-codefresh Digest hasn't updated.
@mikhail-klimko Is it possible that it's some gap in an automation that bumps digests?